runtime.md

```diff
@@ -33,24 +33,79 @@ This is provided so that consumers can find the container's configuration and ro
 }
 ```
 
-## Lifecycle
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
+
+OPEN: we should consider specifying the options/args for each operation.
+For example, checkpoint (for runc) seems to have a lot of params.
```

Comment:
There's a [mailing list thread][1] for this command-line specification, and initial work [in this Gist][2].

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/BIxya5eSNLo
[2]: https://gist.github.com/wking/1d69118ba8b750f85bfc